build.ps1

```diff
@@ -24,14 +21,18 @@ Write-Output "Using MSBuild from: $msbuild"
 if ($lastexitcode -ne 0) { exit $lastexitcode; }
 
 # Create the stable NuGet package
-& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$env:BUILD_NUMBER"
+echo "Creating package"
+& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$date-git$hash"
 if ($lastexitcode -ne 0) { exit $lastexitcode; }
 
 # Create the beta NuGet package
-& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$env:BUILD_NUMBER-beta"
-if ($lastexitcode -ne 0) { exit $lastexitcode; }
+#& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$env:BUILD_NUMBER-beta"
+#if ($lastexitcode -ne 0) { exit $lastexitcode; }
 
 # Copy everything into the output folder
+echo "Copying everything to ./Output"
 Copy-Item "./Xamarin.Essentials/bin/Release" "./Output" -Recurse -Force
 
+get-childitem -recurse .\Output
```

Comment:
now that the build and upload worked, there's no need anymore for additions like this above (and the echo statements), right? then let's remove them, to minimize the changes between this fork and upstream please